src/functions/cr.js

```diff
@@ -0,0 +1,56 @@
+//@flow
+// Row breaks within tabular environments, and line breaks at top level
+
+import defineFunction from "../defineFunction";
+import buildCommon from "../buildCommon";
+import mathMLTree from "../mathMLTree";
+import { calculateSize } from "../units";
+import ParseError from "../ParseError";
+
+defineFunction({
+    type: "cr",
+    names: ["\\\\", "\\cr", "\\newline"],
+    props: {
+        numArgs: 0,
+        numOptionalArgs: 1,
+        argTypes: ["size"],
+        allowedInText: true,
+    },
+
+    handler: (context, args, optArgs) => {
+        return {
+            type: "cr",
+            // \\ and \cr both end the row in a tabular environment
+            newRow: context.funcName !== "\\newline",
```

Comment:
Where is `newRow` used?

Reply:
It's never used. I expected to want it in `environments/array.js`, but that explicitly checks for `\\` or `\cr` without thinking of it as a function, so this isn't needed there.  I could remove it.  Preference?

Reply:
I think we can comment it out, as it can be used in the future.

Reply:
Sounds good. Done.